src/TestableIO.System.IO.Abstractions/IFileSystemInfo.cs

```diff
@@ -8,6 +8,9 @@ public interface IFileSystemInfo
         /// <summary>
         /// Exposes the underlying filesystem implementation. This is useful for implementing extension methods.
         /// </summary>
+        /// <remarks>
+        /// The property is always a global object related to a global current directory.
```

Comment:
I think it would be clearer if you say "**the** global current directory" (as there is always only one).
What do you think?

Reply:
I'm not a native speaker so any suggestions about wording are welcomed! I'll make the change.